microsoft/testsuites/cloud_hypervisor/ch_tests.py

```diff
@@ -39,6 +40,10 @@ def after_suite(self, log: Logger, **kwargs: Any) -> None:
 
     def before_case(self, log: Logger, **kwargs: Any) -> None:
         node = kwargs["node"]
+        if not isinstance(node.os, (CBLMariner, Ubuntu)):
```

Comment:
I am not sure if Skipped Exception is appropriately handled in `before_case`. @squirrelsc can you please confirm?

Reply:
Yes, any exception in before_case will skip the case.